### PR TITLE
Adding comment components to level art entities

### PR DESCRIPTION
**Pull request by mcphedar**
Adding several comment components.

### CODE REVIEW

File: Levels/NewStarbase/NewStarbase.prefab [hunk 1]
```diff
@@ -2535,6 +2535,11 @@
                     "$type": "EditorPendingCompositionComponent",
                     "Id": 16817453197198982078
                 },
+                "Component_[2619917249643206097]": {
+                    "$type": "EditorCommentComponent",
+                    "Id": 2619917249643206097,
+                    "Configuration": "For more information about lighting please see the level design guide in the documentation folder. <Add Link>"
```
Comment (lmbr-pip) [inline Levels/NewStarbase/NewStarbase.prefab:2541]:
```suggestion
                    "Configuration": "For more information about lighting please see the level design guide in the MultiplayerSample documentation folder."
```

File: Levels/NewStarbase/NewStarbase.prefab [hunk 2]
```diff
@@ -4487,6 +4492,11 @@
                     "Id": 7186247374812053450,
                     "Parent Entity": "Entity_[60274689909647]"
                 },
+                "Component_[7963844296059785842]": {
+                    "$type": "EditorCommentComponent",
+                    "Id": 7963844296059785842,
+                    "Configuration": "This is a simple 1x1 tile terrain. The spawners operating within the bounds of the box for the terrain spawner. The child entity height is an image gradient signal that loads a height map asset to generate the height map, which is then sampled by the parent terrain entity, via the terrain height grandient list component. The matching macro color map and macro normal map are loaded in the terrain macro material component. "
```
Comment (lmbr-pip) [inline Levels/NewStarbase/NewStarbase.prefab:4498]:
```suggestion
                    "Configuration": "This is a simple 1x1 tile terrain. The spawners operate within the bounds of the box for the terrain spawner. The child entity height is an image gradient signal that loads a height map asset to generate the height map, which is then sampled by the parent terrain entity, via the terrain height gradient list component. The matching macro color map and macro normal map are loaded in the terrain macro material component."
```

File: Levels/NewStarbase/NewStarbase.prefab [hunk 3]
```diff
@@ -8528,6 +8543,11 @@
                     "$type": "EditorDisabledCompositionComponent",
                     "Id": 15744299950611104828
                 },
+                "Component_[1684972590650734165]": {
+                    "$type": "EditorCommentComponent",
+                    "Id": 1684972590650734165,
+                    "Configuration": "The sun properties of the sky atmosphere contains a  reference to the child sun entity. The transform of the child sun entity determines the position of the sun in the sky atmpshpere. The sky atmosphere is baked into the image based lighting that is loaded into the Global illlumination sample's lighting information from the sky light, the sky atmosphere and the sun entity."
```
Comment (lmbr-pip) [inline Levels/NewStarbase/NewStarbase.prefab:8549]:
```suggestion
                    "Configuration": "The sun properties of the sky atmosphere contain a reference to the child sun entity. The transform of the child sun entity determines the position of the sun in the sky atmosphere. The sky atmosphere is baked into the image-based lighting that is loaded into the Global illumination sample's lighting information from the sky light, the sky atmosphere and the sun entity."
```

File: Levels/NewStarbase/NewStarbase.prefab [hunk 4]
```diff
@@ -30814,7 +30844,7 @@
                 {
                     "op": "replace",
                     "path": "/ContainerEntity/Components/Component_[2563122736071885678]/Transform Data/Rotate/1",
-                    "value": 1.2738307587564779e-12
+                    "value": 1.273830758756478e-12
```
Comment (lmbr-pip) [inline Levels/NewStarbase/NewStarbase.prefab:30847]: nit: Do we know why these changed?

Comment (mcphedar) [reply Levels/NewStarbase/NewStarbase.prefab:30847]: nope. I didn't change anything in the level aside from adding the comment components;

**Review by kberg-amzn — APPROVED** [review]

**Review by lmbr-pip — APPROVED** [review]

**Pull request merged**